`no-magic-numbers` should ignore parseInt radix param

It seems odd to have to make an exception for this every time, or to declare a DEFAULT_RADIX constant for the entire codebase. Not sure what other people think since it is a fairly narrow (though possibly common) case. This is more of a problem with the `radix` rule.

Code:
```
parseInt("1234", 10);
```

Error:
```
  1:18  error  No magic number: 10  no-magic-numbers
```

Config:
```
"no-magic-numbers": 2
"radix": 2
```

Eslint Version: 1.7.0

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27470878-no-magic-numbers-should-ignore-parseint-radix-param?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/lib/rules/no-magic-numbers.js
+++ b/lib/rules/no-magic-numbers.js
@@ -65,9 +65,13 @@ module.exports = function(context) {
             var parent = node.parent,
                 okTypes = detectObjects ? [] : ["ObjectExpression", "Property", "AssignmentExpression"];
 
-            // don't warn on parseInt() radix
+            // don't warn on parseInt() or Number.parseInt() radix
             if (node.parent.type === "CallExpression" && node === node.parent.arguments[1] &&
-                    node.parent.callee.name === "parseInt") {
+                    (node.parent.callee.name === "parseInt" ||
+                    node.parent.callee.type === "MemberExpression" &&
+                    node.parent.callee.object.name === "Number" &&
+                    node.parent.callee.property.name === "parseInt")
+                ) {
                 return;
             }
 

--- a/tests/lib/rules/no-magic-numbers.js
+++ b/tests/lib/rules/no-magic-numbers.js
@@ -25,6 +25,9 @@ ruleTester.run("no-magic-numbers", rule, {
             code: "var x = parseInt(y, 10)"
         },
         {
+            code: "var x = Number.parseInt(y, 10)"
+        },
+        {
             code: "const foo = 42;",
             env: { es6: true }
         },

